src/main/java/com/aws/greengrass/componentmanager/ClientConfigurationUtils.java

```diff
@@ -96,13 +97,15 @@ public static String getGreengrassServiceEndpoint(DeviceConfiguration deviceConf
      * Configure the http client builder with the required certificates for the mutual auth connection.
      *
      * @param deviceConfiguration {@link DeviceConfiguration}
+     * @param securityService {@link SecurityService}
      * @return configured http client
      */
-    public static ApacheHttpClient.Builder getConfiguredClientBuilder(DeviceConfiguration deviceConfiguration) {
```

Comment:
this is breaking all plugins, do not change this

Reply:
reverted